clc-stackage.cabal

```diff
@@ -1253,7 +1255,6 @@ library
              kmeans ==0.1.3,
              knob ==0.2,
              koji ==0.0.2,
-             krank ==0.2.3,
```

Comment:
```
Failed to build krank-0.2.3.
Build log (
~/.cabal/logs/ghc-9.2.4/krank-0.2.3-663c8ab064a7696c9ba32fe0824fd072e2172b48235b529ec2013543238cb6ca.log
):
Configuring library for krank-0.2.3..
Preprocessing library for krank-0.2.3..
Building library for krank-0.2.3..
[ 1 of 11] Compiling Krank.Types      ( src/Krank/Types.hs, dist/build/Krank/Types.o, dist/build/Krank/Types.dyn_o )
[ 2 of 11] Compiling Krank.Checkers.Ignore ( src/Krank/Checkers/Ignore.hs, dist/build/Krank/Checkers/Ignore.o, dist/build/Krank/Checkers/Ignore.dyn_o )

<no location info>: warning: [-Wmissed-extra-shared-lib]
    libz.so: cannot open shared object file: No such file or directory
    It's OK if you don't want to use symbols from it directly.
    (the package DLL is loaded by the system linker
     which manages dependencies by itself).

<no location info>: warning: [-Wmissed-extra-shared-lib]
    libpcre.so: cannot open shared object file: No such file or directory
    It's OK if you don't want to use symbols from it directly.
    (the package DLL is loaded by the system linker
     which manages dependencies by itself).

src/Krank/Checkers/Ignore.hs:35:31: error:
    • Exception when trying to run compile-time code:
        src/PyF/Internal/Meta.hs:(212,1)-(347,76): Non-exhaustive patterns in function translateTHtoGHCExt

      Code: template-haskell-2.18.0.0:Language.Haskell.TH.Quote.quoteExp
              fmt
              "Impossible case, update the guard with: {ByteString.unpack command}"
    • In the quasi-quotation:
        [fmt|Impossible case, update the guard with: {ByteString.unpack command}|]
   |
35 |           | otherwise = error [fmt|Impossible case, update the guard with: {ByteString.unpack command}|]
   |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```